test/generic/test_object_storage.py

```diff
@@ -4,24 +4,32 @@
 
 
 def test_clear_objects():
-    obj_storage = object_storage.ObjectStorage()
+    """
+    Asserts that objects in storage are cleared
+    """
+    obj_storage = (
+        object_storage.ObjectStorage()
+    )  #  obj_storage is a wrapper object to a collection of objects
 
     x = torch.tensor(1)
     obj_storage.set_obj(x)
 
-    objs = obj_storage.current_objects()
+    objs = obj_storage.current_objects()  # Returns a copy of the objects in obj_storage(here:x)
 
     assert len(objs) == 1
     assert objs[x.id] == x
 
-    ret_val = obj_storage.clear_objects()
+    ret_val = obj_storage.clear_objects()  # Completely removes all objects from obj_storage
 
     objs = obj_storage.current_objects()
     assert len(objs) == 0
     assert ret_val == obj_storage
 
 
 def test_clear_objects_return_None():
+    """
+    Asserts that objects are cleared and returned as none
+    """
```

Comment:
For example here I think it could be:
> Check storage clear_objects method when no return is needed

More generally, the docstring should resume the main idea of the function in a clear way: it's a helper. Try to ba as clear and explicit as possible when you add dosctring ;)